aten/src/ATen/native/TensorConversions.cpp

```diff
@@ -756,9 +785,97 @@ Tensor dense_to_sparse_bsr(const Tensor& self, IntArrayRef blocksize) {
 }
 
 Tensor dense_to_sparse_bsc(const Tensor& self, IntArrayRef blocksize) {
-  AT_ERROR(
-      "Conversion from ", self.layout(), " to SparseBsc is currently not supported.");
-  return self;
+  TORCH_CHECK(
+      blocksize[0] > 0 && blocksize[1] > 0,
+      "blocksize needs to be non zero, but got ",
```

Comment:
The requirement is `>0` I don't think we allow blocksize to be 0. That requirement was copied here from the BSR conversion though I am not sure if that requirement is coming from an invariant or just to make it easier to avoid divide-by-zero later on.

Reply:
ugh sorry yeah, I meant to suggest > 0 rather than just != 0, my bad